indico/web/client/js/react/forms/validators.js

```diff
@@ -68,7 +68,7 @@ function optional(arg = null) {
     return chain(optional(), arg);
   }
   return value => {
-    if (value === null) {
+    if ([null, undefined].includes(value)) {
```

Comment:
Not sure if `includes()` really makes things more readable here.. I'd go for good old `... || ...` here.

Reply:
Yea, sounds good!